tests/Vesting.test.ts

```diff
@@ -143,5 +145,61 @@ describe("Vesting service", async () => {
 			expect(tomMoney[1].value.lovelace).toBeTypeOf('bigint');
 
 		});
+		it("sasha can cancel", async (context: localTC) => {
+		    const {h, h: { network, actors, delay, state }} = context;
+			const { sasha, tom, pavel } = actors;
+
+			const tx1 = new Tx();
+			const sashaMoney = await sasha.utxos;
+
+			tx1.addInput(sashaMoney[0]);
+			tx1.addOutput(new TxOutput(sasha.address, new Value(3n * ADA)));
+			tx1.addOutput(new TxOutput(sasha.address, new Value(3n * ADA)));
+			tx1.addOutput(new TxOutput(sasha.address, new Value(3n * ADA)));
+			tx1.addOutput(
+				new TxOutput(
+				sasha.address,
+				new Value(sashaMoney[0].value.lovelace - 11n * ADA)
+				)
+			);
+
+			await h.submitTx(tx1);
+
+			const v = new Vesting(context);
+			const t = BigInt(Date.now());
+			const d = t + BigInt(2*60*60*1000);
```

Comment:
what this value should be?

Reply:
I added 2 checks. Indeed time looks like SLT. The type in the Datum is IntData, but in the main() there is Time.
Time should be inferred from BigInt 
`Time::new(millis_since_1970: Int) -> Time`

https://www.hyperion-bt.org/helios-book/lang/builtins/time.html?highlight=Time#time